app.py

```diff
@@ -7,37 +7,41 @@
 import sys
 import yaml
 
+
 def configure():
-  defaults = {  "aws": {
-                  "region": "us-east-1",
-                  "ec2_instance_type": "c5n.large",
-                  "asg_size": 2,
-                  "max_spot_price": "",
-                }
-             }
-  try:
-    with open("config.yaml") as f:
-      user_config = yaml.safe_load(f)
-  except yaml.YAMLError as exc:
-    print("Failed parsing the configuration file, exiting", exc)
-    sys.exit(1)
-  except OSError as exc:
-    print("Unable to read configuration file, exiting", exc)
-    sys.exit(1)
-  # Deep merging should be done better. The moment we need more
-  # parameters, this will need to be fixed
-  config = { "aws" : {**defaults['aws'], **user_config['aws']}}
-  return config
+    defaults = {"aws": {
+        "region": "us-east-1",
+        "ec2_instance_type": "c5n.large",
+        "asg_size": 2,
+        "max_spot_price": "",
+    }
+    }
+    try:
+        with open("config.yaml") as f:
+            user_config = yaml.safe_load(f)
+    except yaml.YAMLError as exc:
+        print("Failed parsing the configuration file, exiting", exc)
+        sys.exit(1)
+    except OSError as exc:
+        print("Unable to read configuration file, exiting", exc)
+        sys.exit(1)
+    # Deep merging should be done better. The moment we need more
+    # parameters, this will need to be fixed
+    config = {"aws": {**defaults['aws'], **user_config['aws']}}
+    return config
+
 
 def cdk_init(config: dict):
-  app = core.App()
-  vpc_stack = FoldingVpcStack(app, "folding-vpc", env=core.Environment(region=config['aws']['region']))
-  asg_stack = FoldingAsgStack(app, "folding-asg", region=config['aws']['region'], vpc=vpc_stack.vpc, 
+    app = core.App()
+    vpc_stack = FoldingVpcStack(app, "folding-vpc", env=core.Environment(region=config['aws']['region']))
+    asg_stack = FoldingAsgStack(app, "folding-asg", region=config['aws']['region'], vpc=vpc_stack.vpc,
```

Comment:
since we're using different editors, I see the indentation has change, Might I suggest adding an `.editorconfig`?

https://editorconfig.org/

Reply:
Works for me! Do you want to check in your config file, so that I can rebase my branch and work with that?